galsim/gsobject.py

```diff
@@ -1884,6 +1881,7 @@ def drawKImage(self, image=None, nx=None, ny=None, bounds=None, scale=None,
                             If `scale` is None and `image` is None, then use the Nyquist scale.
                             If `scale <= 0` (regardless of `image`), then use the Nyquist scale.
                             [default: None]
+        @param wcs          The wcs. [default: None]
```

Comment:
I took the liberty of reverting this (and the InterpotedKImage wcs option) to make my life easier merging in the new stuff that just got merged to master.  The commit is 44da24d823d401966e182688d54e86141c1c70af if you want to look through and make sure I didn't remove anything you think is important.  

Nothing broke in the unit tests when I did that (other than the unit tests specifically designed to test these features, which I removed).  I also ran your test_CRG and test_CRG_noise scripts, which also seem to be ok.